grafana/grafana
# InfluxDB: variables migration to backend

**What this PR does / why we need it**:
Part of https://github.com/grafana/grafana/issues/43076

This PR contains the changes needed to applyTemplateVariables() to InfluxQL queries (which are run through the backend).

**Test scenarios**:

1. Run make devenv sources=influxdb,influxdb1,grafana grafana_version=8.2.4 (or other version before changes)
2. Open up http://localhost:3000/explore && http://127.0.0.1:3001/explore (to test older 8.2.4 version)
3. Select gdev-influxdb-influxql
    - applyTemplateVariables() should apply all template variables in explore queries that are run through the backend via `api/ds/query`
4. Select gdev-influxdb1-influxql
    - Should behave like `gdev-influxdb-influxql`

PS: make sure 
[feature_toggles]
enable = influxdbBackendMigration
